vnext/Microsoft.ReactNative/Modules/DevSettingsModule.h

```diff
@@ -5,30 +5,30 @@
 #include <cxxreact/CxxModule.h>
 #include <functional/functor.h>
 
-namespace Microsoft::ReactNative {
-
-class DevSettingsModule : public facebook::xplat::module::CxxModule {
- public:
-  DevSettingsModule(Mso::VoidFunctor &&reload);
+#include <NativeModules.h>
+#include <ReactHost/React.h>
 
-  static const char *name;
+namespace Microsoft::ReactNative {
 
-  std::string getName() override;
-  std::map<std::string, folly::dynamic> getConstants() override;
-  std::vector<Method> getMethods() override;
+REACT_MODULE(DevSettings)
+struct DevSettings {
+  REACT_INIT(Initialize) void Initialize(winrt::Microsoft::ReactNative::ReactContext const &reactContext) noexcept;
 
-  void reload();
-  void setHotLoadingEnabled(bool isHotLoadingEnabled);
-  void setIsDebuggingRemotely(bool isDebuggingRemotelyEnabled);
-  void setLiveReloadEnabled(bool setLiveReloadEnabled);
-  void setProfilingEnabled(bool setProfilingEnabled);
-  void toggleElementInspector();
+  REACT_METHOD(reload) void reload() noexcept;
+  REACT_METHOD(reloadWithReason) void reloadWithReason(std::string reason) noexcept;
+  REACT_METHOD(onFastRefresh) void onFastRefresh() noexcept;
+  REACT_METHOD(setHotLoadingEnabled) void setHotLoadingEnabled(bool isHotLoadingEnabled) noexcept;
+  REACT_METHOD(setIsDebuggingRemotely) void setIsDebuggingRemotely(bool isDebuggingRemotelyEnabled) noexcept;
+  REACT_METHOD(setProfilingEnabled) void setProfilingEnabled(bool isProfilingEnabled) noexcept;
+  REACT_METHOD(toggleElementInspector) void toggleElementInspector() noexcept;
+  REACT_METHOD(addMenuItem) void addMenuItem(std::string title) noexcept;
+  REACT_METHOD(setIsShakeToShowDevMenuEnabled) void setIsShakeToShowDevMenuEnabled(bool enabled) noexcept;
 
-  // iOS only.
-  void setIsShakeToShowDevMenuEnabled(bool enabled);
+  static void SetReload(Mso::React::ReactOptions const &options, Mso::VoidFunctor &&func) noexcept;
 
  private:
-  Mso::VoidFunctor m_reload;
+  static winrt::Microsoft::ReactNative::IReactPropertyName ReloadProperty() noexcept;
```

Comment:
>winrt::Microsoft::ReactNative::IReactPropertyName [](start = 9, length = 49)

Since this method is private, consider using the ReactPropertyId instead to have a strongly typed access.